src/components/dependencies/RescoringModal.js

```diff
@@ -1965,7 +2092,7 @@ const Rescore = ({
           {
             customRescoringsWithoutComment.map((r, idx) => <Typography key={idx} variant='body2'>
               {
-                `${r.finding.package_name} - ${type === artefactMetadataTypes.LICENSE ? r.finding.license.name : r.finding.cve}`
+                r.finding.finding_type
```

Comment:
I suggest to state a more specific hint here, e.g. the type specific portion of the rescoring identity.